src/modules/45-projects-orgs/components/ProjectSelector/ProjectSelector.tsx

```diff
@@ -71,8 +72,9 @@ const RenderColumnMenu: Renderer<CellProps<ProjectAggregateDTO>> = ({ row }) =>
   )
 }
 
-const ProjectSelect: React.FC<ProjectSelectorProps> = ({ onSelect }) => {
-  const { accountId } = useParams<AccountPathProps>()
+const ProjectSelect: React.FC<ProjectSelectorProps> = ({ onSelect, fallbackAccountId = '' }) => {
+  const accountProps = useParams<AccountPathProps>()
+  const accountId = accountProps.accountId || fallbackAccountId
```

Comment:
100% not an issue, but you can combine these by just giving the destructuring a default.
```suggestion
  const { accountId = fallbackAccountId } = useParams<AccountPathProps>()
```